<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Title</class>
 <widget class="QWidget" name="Title">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnMin">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/min.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/min_hover.png);
}
QPushButton:pressed
{
image: url(:/images/min_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/close.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/close_hover.png);
}
QPushButton:pressed
{
image: url(:/images/close_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMax">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/max.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/max_hover.png);
}
QPushButton:pressed
{
image: url(:/images/max_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29, 121, 184);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>5</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 200 italic 9pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Online Course</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetting">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/setting.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/setting_hover.png);
}
QPushButton:pressed
{
image: url(:/images/setting_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/home_pressed.png);
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/course.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/course_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/discuss.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/discuss_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDownload">
   <property name="geometry">
    <rect>
     <x>1149</x>
     <y>40</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/download.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/download_hover.png);
}
QPushButton:pressed
{
image: url(:/images/download_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFriend">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>40</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/friend.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/friend_hover.png);
}
QPushButton:pressed
{
image: url(:/images/friend_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUser">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>40</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image: url(:/images/user.png);
border-style:outset;
}
QPushButton:hover
{
image: url(:/images/user_hover.png);
}
QPushButton:pressed
{
image: url(:/images/user_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btnMin</zorder>
  <zorder>btnClose</zorder>
  <zorder>btnMax</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>btnSetting</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>btnDownload</zorder>
  <zorder>btnFriend</zorder>
  <zorder>btnUser</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
